src/components/Dashboard/TodoList.vue

```diff
@@ -912,7 +918,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
 
     affiliationInvitations.forEach(affiliationInvitation => {
       // only active (pending) affiliation invitations are to be converted into todo item for now
-      if (affiliationInvitation.type === 'RequestAccess' && affiliationInvitation.status === 'ACTIVE') {
+      if (affiliationInvitation.type === 'REQUEST' && affiliationInvitation.status === 'PENDING') {
```

Comment:
Please create a type (interface) for `affiliationInvitation` and use it where appropriate (eg, line 917 above).

In that type, please create an enum for `type` so that we don't use untyped strings. Also create an enum for `status` if it doesn't make sense to reuse the FilingStatus enum.